src/components/pages/public/sign-up/index.js

```diff
@@ -43,12 +43,16 @@ class SignUp extends Component {
         } = this.props;
 
         return (
-            <form noValidate onSubmit={handleSubmit((user) => { createAccount(user); })}>
+            <form noValidate onSubmit={handleSubmit(user => createAccount(user))}>
                 <section className={this.getClassName()} style={style}>
                     <Link to="/home">
                         <img src="/assets/rainbow-logo.svg" alt="rainbow logo" className="rainbow-auth-firebase-signup_image" />
                     </Link>
-                    <p className="rainbow-auth-firebase-signup_header">Sign up</p>
+                    <p className="rainbow-auth-firebase-signup_header">Create Account</p>
+                    <p className="rainbow-auth-firebase-signup_have-account">
+                        Already have an account?
+                        <Link className="rainbow-auth-firebase-signup_link" to="/home/signin"> Login here</Link>
```

Comment:
you need to update with master and use FormattedMessage here and add this words to i18n dictionary in order to translate it